README.md

```diff
@@ -1,3 +1,28 @@
 # Documentation Uploader for Client Libraries
 
-This tool uploads documentation for publishing to googleapis.dev.
+This tool uploads documentation for publishing to googleapis.dev
+and cloud.google.com (see
+[`doc-templates`](https://github.com/googleapis/doc-templates) and
+[`doc-pipeline`](https://github.com/googleapis/doc-pipeline) for more info).
+
+## Installation
+
+You can install `docuploader` with:
+
+```
+pip install gcp-docuploader
+```
+
+## Usage
+
+There are two steps for uploading docs:
+1. Create a metadata file. See [`metadata.proto`](./docuploader/protos/metadata.proto)
+   for the format.
+   * You can use `docuploader create-metadata` to create a `docs.metadata` file.
+   * Alternatively, you can create a `docs.metadata.json` file independently.
+1. Upload the docs with `docuploader upload`.
```

Comment:
Most of the time we use this it it followed by a lot of options. But others could be simply using `docuploader upload .`, what do you think about adding the usage notes?

```
Usage: docuploader upload [OPTIONS] DOCUMENTATION_PATH

Options:
  --staging-bucket TEXT      The bucket to upload the staged documentation to.
  --credentials TEXT         Path to the credentials file to use for Google
                             Cloud Storage.

  --metadata-file TEXT       Path to the docs.metadata file.
  --destination-prefix TEXT  Prefix to include when uploading tar file. A -
                             will be added after the prefix, if there is one.

  --help                     Show this message and exit.
```

Reply:
I agree - providing all the possible options like this would be helpful

Reply:
Done.